.github/settings.yml

```diff
@@ -0,0 +1,68 @@
+---
+repository:
+  name: openshift
+  description: OpenShift catalog templates for ownCloud
+  topics: docker, openshift
+
+  private: false
+  has_issues: true
+  has_projects: false
+  has_wiki: false
+  has_downloads: false
+
+  default_branch: master
+
+  allow_squash_merge: true
+  allow_merge_commit: true
+  allow_rebase_merge: true
+
+labels:
+  - name: bug
+    color: d73a4a
+    description: Something isn't working
+  - name: documentation
+    color: 0075ca
+    description: Improvements or additions to documentation
+  - name: duplicate
+    color: cfd3d7
+    description: This issue or pull request already exists
+  - name: enhancement
+    color: a2eeef
+    description: New feature or request
+  - name: good first issue
+    color: 7057ff
+    description: Good for newcomers
+  - name: help wanted
+    color: 008672
+    description: Extra attention is needed
+  - name: invalid
+    color: e4e669
+    description: This doesn't seem right
+  - name: question
+    color: d876e3
+    description: Further information is requested
+  - name: wontfix
+    color: ffffff
+    description: This will not be worked on
+
+teams:
+  - name: bot
+    permission: admin
+
+branches:
+  - name: master
+    protection:
+      required_pull_request_reviews:
+        required_approving_review_count: 1
+        dismiss_stale_reviews: false
+        require_code_owner_reviews: false
+        dismissal_restrictions: {}
+      required_status_checks:
+        strict: false
+        contexts:
+          - continuous-integration/drone/pr
+      enforce_admins: false
```

Comment:
Do you remember the outcome of our discussion for the oCIS repo?
There we set `enforce_admins: null` (https://github.com/owncloud/ocis/commit/ce9177e21ceea8e23710774f09efce49424e2422)
But also some more changes have been applied:
https://github.com/owncloud/ocis/commit/7b7fb647a10e8d0aa8d4b2e29a32dff6d884db85
https://github.com/owncloud/ocis/commit/012741d44f286ace2beac9f352673743b5c5509e

Reply:
Right, fixed. The changed description should not matter.